compiler/rustc_codegen_llvm/src/back/lto.rs

```diff
@@ -200,18 +200,11 @@ pub(crate) fn run_fat(
     modules: Vec<FatLtoInput<LlvmCodegenBackend>>,
     cached_modules: Vec<(SerializedModule<ModuleBuffer>, WorkProduct)>,
 ) -> Result<LtoModuleCodegen<LlvmCodegenBackend>, FatalError> {
-    let diag_handler = cgcx.create_diag_handler();
-    let (symbols_below_threshold, upstream_modules) = prepare_lto(cgcx, &diag_handler)?;
+    let dcx = cgcx.create_dcx();
```

Comment:
I wonder if this should be `create_diag_ctxt` or something

Reply:
I don't see why function names need to be different to variable names. Consistency is good.

Reply:
It's consistent with variable names, but not the type name. I consider it more normal for a constructor function to be called create_type_name, personally.